airflow/api_fastapi/core_api/routes/public/assets.py

```diff
@@ -63,9 +63,31 @@ def get_assets(
         limit=limit,
         session=session,
     )
+    assets_select.options(subqueryload(AssetEvent.created_dagruns))
```

Comment:
I don't think this is the original loading options from the legacy endpoint.

Reply:
My bad. Took from wrong endpoint. Let me correct it

Reply:
It cant be this, it has to be same like the `get_asset` endpoint